ROCK/GPIOBase.py

```diff
@@ -276,7 +300,7 @@ def fn_event_detect(self, channel, ctx):
             file.seek(0)  # reset read cursor
         print("unregistering add_event_detect for {}".format(channel))
 
-    def add_event_detect(self, channel, edge, callback, bouncetime):
+    def add_event_detect(self, channel, edge, callback, bouncetime = 1):
```

Comment:
Any reason to put a default value? Curious

Reply:
It's been a while since I wrote this, so I'm not 100% sure, but if I remember correctly, I added this because some program/library I was trying to run didn't include a bouncetime parameter and it was causing issues